ml-fundamentals/ml-problem-formulation-envisioning.md

```diff
@@ -0,0 +1,115 @@
+# Envisioning & Problem formulation
+
+Before beginning a data science investigation, it is necessary to have first defined problem statement which the data science team can explore; the manner of this definition can have a significant influence on whether the project is likely to be successful.  
+
+The main goal of envisioning is to work alongside subject matter experts to formulate the problem in such a way that there is a shared understanding of the problem domain, a clear goal, and a predefined approach to evaluating a potential solution; this is often done through a series of 'envisioning' sessions.
+
+## Understanding the problem domain
+
+Generally, before defining a project scope for a data science investigation, we must first understand the problem domain:
+
+* What is the problem?
+* Why does the problem need to be solved?
+* Does this problem require a machine learning solution?
+* How would a potential solution be used?
+
+However, establishing this understanding can prove difficult, especially for those unfamiliar with the problem domain. To ease this process, we can approach problems in a structured way by taking the following steps:  
+
+* Identify a measurable problem and define this in business terms, such that there is a clear objective and a good understanding of the input elements that can be controlled which may affect this objective; here, it is best to be as specific as possible.
+* Decide how the performance of a solution should be measured, such that it is aligned with the business objective, and the data needed to evaluate the effectiveness of a solution. Note that the data required to evaluate a solution's impact on the overall objective is not necessarily the same as that which may be required to create a solution.
```

Comment:
I am finding it a little bit hard to follow this sentence as well, can it be formulated a bit more concisely? How is the performance aligned to the data needed. and what does it mean that the data required to evaluate a solutions impact on the overall objective...

I have made a suggestion, but again, not sure if I got it correctly.